projects/github-actions/repo-gardening/src/tasks/check-description/index.js

```diff
@@ -446,6 +479,64 @@ async function updateLabels( payload, octokit ) {
 	}
 }
 
+/**
+ * Based off the changelog entries committed with the PR,
+ * let's attempt to infer the type of the PR.
+ * If we can infer a type, we'll add a label to the PR, matching that type.
+ * If we cannot, we'll leave the PR as is.
+ *
+ * @param {WebhookPayloadPullRequest} payload          - Pull request event payload.
+ * @param {GitHub}                    octokit          - Initialized Octokit REST client.
+ * @param {Array}                     changelogEntries - List of changelog entries.
+ *
+ * @return {Promise<boolean>} Promise resolving to boolean ; true if we were able to infer a type, false otherwise.
+ */
+async function inferType( payload, octokit, changelogEntries ) {
+	const {
+		repository: {
+			owner: { login: ownerLogin },
+			name: repo,
+		},
+		pull_request: { number },
+	} = payload;
+
+	// Extract the type of the PR from the changelog entries.
+	let type = '';
+	for ( const entry of changelogEntries ) {
+		if ( entry.changeType ) {
+			type = entry.changeType;
+			break;
+		}
+	}
+	if ( ! type ) {
+		return false;
+	}
+
+	// Mapping between the different changelogger types
+	// (default and custom ones created for the Jetpack plugin)
+	// and the labels we use.
+	const typeMapping = {
+		Bug: [ 'security', 'fixed', 'bugfix' ],
+		Task: [ 'added', 'major' ],
+		Enhancement: [ 'changed', 'enhancement' ],
+		Janitorial: [ 'deprecated', 'removed', 'compat', 'other' ],
```

Comment:
'other' could be anything, its only defining characteristic is "not worth listing on wporg".

Reply:
True. I'll remove it in eaa782bb772021a237b3ae3ae398b60e4112d1b9